packages/core/package.json

```diff
@@ -132,6 +132,8 @@
     "react-router-dom": "^5.0.1",
     "react-transition-group": "^2.5.0",
     "react-window": "^1.8.4",
+    "smartcharts-beta": "^0.6.2",
+    "tt-react-custom-scrollbars": "^4.2.1-tt2",
```

Comment:
where do we use scrollbars outside of components?

Reply:
removed it